ontology/yaml/resources/HVAC/entity_types/AION.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+
+AION_BS_SS:
+  description: "Basic air ionizer with start/stop"
+  is_canonical: true
+  implements:
+  - AION
+  - SS
+
+AION_BS_SS_SRC_TA_FA:

```

Comment:
should be AION_SS_SRC if that is all that it is; the other abbreviations aren't defined as part of the abstract types.

Reply:
ok sir. will do as you say.